app/(main)/markets/MarketDetails.tsx

```diff
@@ -78,8 +85,34 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
                 />
                 <h1 className="text-xl font-semibold">{fixedProductMarketMaker.title}</h1>
               </div>
-              <div className="!mt-7">
+              <div className="!mt-7 space-y-4">
                 <OutcomeBar market={fixedProductMarketMaker} />
+                <div className="flex items-center justify-end space-x-8">
+                  <div className="flex items-center space-x-2 text-sm font-semibold text-text-med-em">
+                    <TokenLogo
+                      address={fixedProductMarketMaker.collateralToken}
+                      size="xs"
+                      className="size-4"
+                    />
+                    <p>
+                      {formatTokenPrice(fixedProductMarketMaker.collateralVolume)}
```

Comment:
Let's use `scaledCollateralVolume` with `formatValueWithFixedDecimals` here.
```suggestion
                      {formatValueWithFixedDecimals(fixedProductMarketMaker.scaledCollateralVolume)}
```